packages/extras/src/lib/interactivity/setupInteractivity.ts

```diff
@@ -217,12 +237,14 @@ export const setupInteractivity = (context: InteractivityContext) => {
     DOM_EVENTS.forEach(([eventName]) => {
       target.removeEventListener(eventName, getEventHandler(eventName))
```

Comment:
FYI there's a minor leak here - removeEventListener removes handlers by reference, but getEventHandler returns a new function every time it is called, so it will never match the previously added handler. 

Probably inconsequential as the canvas is not often unmounted, but I could send a fix if you're interested

Reply:
Will do :+1:

Reply:
Fixed in https://github.com/threlte/threlte/pull/1526